mods/seasonal/moves.js

```diff
@@ -265,6 +289,60 @@ exports.BattleMovedex = {
 		target: "normal",
 		type: "Steel",
 	},
+	// Cerberax
+	blimpcrash: {
+		accuracy: 100,
+		basePower: 120,
+		category: "Physical",
+		id: "blimpcrash",
+		isNonstandard: true,
+		isViable: true,
+		name: "Blimp Crash",
+		pp: 10,
+		priority: 0,
+		flags: {contact: 1, protect: 1, mirror: 1, nonsky: 1},
+		volatileStatus: 'smackdown',
+		recoil: [1, 2],
+ 		onPrepareHit: function (target, source) {
+ 			this.attrLastMove('[still]');
+ 			this.add('-anim', source, "Head Smash", target);
+ 		},
+		effect: {
```

Comment:
If the volatile status you're applying is Smack Down, you don't need to include an effect. It'll use Smack Down's effect data, and not any of this code.

Reply:
Good to know, updated